server/config/config.go

```diff
@@ -99,11 +99,23 @@ const (
 	// DefaultAllowUnprotectedTxs value is false
 	DefaultAllowUnprotectedTxs = false
 
+	// DefaultBatchRequestLimit is the default maximum batch request limit.
+	DefaultBatchRequestLimit = 1000
+
+	// DefaultBatchResponseMaxSize is the default maximum batch response size.
+	DefaultBatchResponseMaxSize = 25 * 1000 * 1000
```

Comment:
do these need to be configurable via the `toml` template?

Reply:
yes, it's better to be configurable in [toml](https://github.com/MANTRA-Chain/mantrachain-e2e/pull/2/files#diff-ce01c603a64da97da1ca690b8ae60f99c1becb792b75dff87c24168b5d356313R7) or via flag